Gemfile

```diff
@@ -42,6 +42,8 @@ gem 'rack-rewrite'
 gem 'roadie', '~> 2.4.0' # version for rails 3
 gem 'mechanize'
 
+gem 'hirb', require: false
```

Comment:
toto je pre mna must have gem pri pozerani veci v database cez konzolu. a to pozeram aj v produkcii. loadne sa to v hooku, ktory bezi len pri konzole.